src/test/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementServiceTest.java

```diff
@@ -140,7 +138,8 @@ void createConfirmationStatement() throws ServiceException, CompanyNotFoundExcep
     }
 
     @Test
-    void createPayableResourceConfirmationStatement() throws ServiceException, CompanyNotFoundException {
+    void createPayableResourceConfirmationStatementTest() throws ServiceException, CompanyNotFoundException {
+        // Arrange
```

Comment:
If you're adding these, I would suggest using "Given", "When" & "Then" which tend to be used elsewhere